beets/util/artresizer.py

```diff
@@ -77,7 +77,10 @@ def pil_resize(maxwidth, path_in, path_out=None, quality=0, max_filesize=0):
             # Use PIL's default quality.
             quality = -1
 
-        im.save(util.py3_path(path_out), quality=quality)
+        # progressive=False only affects JPEG writter and is the default
+        # leaving for explicitness
```

Comment:
Maybe this comment could be a little more obvious for posterity—that is, something like:

> `progressive=False` only affects JPEGs and is the default, but we include it here for explicitness.

Reply:
fixed